core-services/image-mirroring/openshift/mapping_origin_4_7

```diff
@@ -173,6 +173,8 @@ registry.svc.ci.openshift.org/ocp/4.7:oc-rpms quay.io/openshift/origin-oc-rpms:4
 registry.svc.ci.openshift.org/ocp/4.7:openshift-apiserver quay.io/openshift/origin-openshift-apiserver:4.7 quay.io/openshift/origin-openshift-apiserver:4.7.0
 registry.svc.ci.openshift.org/ocp/4.7:openshift-controller-manager quay.io/openshift/origin-openshift-controller-manager:4.7 quay.io/openshift/origin-openshift-controller-manager:4.7.0
 registry.svc.ci.openshift.org/ocp/4.7:openshift-state-metrics quay.io/openshift/origin-openshift-state-metrics:4.7 quay.io/openshift/origin-openshift-state-metrics:4.7.0
+registry.svc.ci.openshift.org/ocp/4.7:openstack-cinder-csi-driver quay.io/openshift/origin-openstack-cinder-csi-driver:4.7 quay.io/openshift/origin-openstack-cinder-csi-driver:4.7.0
```

Comment:
Why would we have `csi-driver-manilla` and `openstack-cinder-csi-driver`?  This should be `csi-driver-cinder`

Reply:
The convention is split already, aws-ebs-csi-driver, ovirt-csi-driver vs csi-driver-manila, csi-driver-nfs

Reply:
At some moment we decided to call all csi images like `<vendor>-<project>-csi-driver` (like `aws-ebs-csi-driver`). It means that we eventually should create an alias for  `csi-driver-manila` and call it `openstack-manila-csi-driver`. But this will be done in another patch.
So far, `openstack-cinder-csi-driver` complies with the naming convention.

Reply:
Ok, that's fine.